.github/workflows/auto-update-translator-cid.yml

```
@@ -84,7 +84,7 @@ jobs:
         if: ${{ env.TARGET_PRID == null }}
         run: |
           env
-          ./scripts/check_update_tranlator_cid.sh $CID_LATEST $CID_CURRENT
+          ./scripts/check-update-tranlator-cid.sh $CID_LATEST $CID_CURRENT
```

```suggestion
          ./scripts/check-update-translator-cid.sh $CID_LATEST $CID_CURRENT
```
Also please rename the script accordingly.

---

Done.